go.mod

```diff
@@ -26,6 +28,7 @@ require (
 	github.com/hashicorp/terraform-plugin-log v0.9.0
 	github.com/hashicorp/terraform-plugin-testing v1.4.0
 	github.com/secure-systems-lab/go-securesystemslib v0.7.0
+	github.com/sigstore/cosign v1.13.2
```

Comment:
This is pulling in two versions of cosign, we probably don't mean to do that.

Reply:
Right you are! Fixed.